org-formation/organization.yaml

```diff
@@ -567,6 +568,21 @@ Organization:
         budget-alarm-threshold: 1000
         budget-alarm-threshold-email-recipient: aws-genie-prod@sagebase.org
 
+  SchematicProdAccount:
+    Type: OC::ORG::Account
+    Properties:
+      AccountName: org-sagebase-schematic-prod
+      RootEmail: aws-schematic-prod@sagebase.org
+      Alias: org-sagebase-schematic-prod
+      Tags:
+        <<: !Include ./_default_org_tags.yaml
+        Department: FAIR
+        Project: DCA
+        AccountOwner: milen.nikolov@sagebase.org
+        CostCenter: "TBD"
```

Comment:
@brucehoff @milen-sage please check Department/Project/AccountOwner and CostCenter are correct.